<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="accessibleName">
   <string>mainWindow</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">#centralWidget{
background-image:url(:/images/images/P FF.png);
}

QWidget{
border-style: solid;
border-color:  black;
border-width: 1px;
}</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>7</x>
      <y>530</y>
      <width>70</width>
      <height>67</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">#pushButton { 
background-color: transparent;
border-image: url(:/iconos/images/Iconos/Menu_azul.png); 
background: none; 
border: none; 
background-repeat: none;
 } 
#pushButton:pressed {
border-image: url(:/iconos/images/Iconos/Menu_azul.png); 
 }
#pushButton:checked{
 border-image: none;
 }</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="bottom_bar">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>529</y>
      <width>766</width>
      <height>70</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">#bottom_bar { 
background-color: transparent;
border-image: url(:/barras/images/Barras/Parametros_azul.png); background: none; 
border: none; 
background-repeat: none;
 } </string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_fisicos">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>560</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">#pb_fisicos { 
background-image: url(:/iconos/screen800x600/iconos/Medidor azul.png);
border: none; 
background-repeat: none;
background-position: center;
} </string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_quimicos">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>560</y>
      <width>71</width>
      <height>41</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">#pb_quimicos { 
background-image: url(:/iconos/screen800x600/iconos/Matraz azul.png);
border: none; 
background-repeat: none;
background-position: center;
} </string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_electricos">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>560</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">#pb_electricos { 
background-image: url(:/iconos/screen800x600/iconos/Clavija azul.png);
border: none; 
background-repeat: none;
background-position: center;
} </string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="active_param_label">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>82</x>
      <y>580</y>
      <width>341</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Phetsarath OT</family>
      <pointsize>16</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">#active_param_label { 
background-color: transparent;
border: none; 
background-repeat: none;
 } </string>
    </property>
    <property name="text">
     <string>Par√°metros</string>
    </property>
    <property name="textFormat">
     <enum>Qt::PlainText</enum>
    </property>
   </widget>
   <widget class="QLabel" name="top_bar">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>2</y>
      <width>759</width>
      <height>27</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">#top_bar { 
background-color: transparent;
background-image: url(:/barras/screen800x600/barras/Indice diagrama limpio.png);
border: none; 
background-repeat: none;
 } </string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="asa_logo">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>550</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">#asa_logo{ 
background-image: url(:/iconos/screen800x600/iconos/Logo.png);
border: none; 
background-repeat: none;
background-position: center;
} </string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_hora">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>50</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Hora</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_dia">
    <property name="geometry">
     <rect>
      <x>245</x>
      <y>28</y>
      <width>291</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Dia</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="prof_pic">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>10</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">#prof_pic { 
background-image: url(:/iconos/screen800x600/iconos/Prof pic azul.png);
border: none; 
background-repeat: none;
background-position: center;
} </string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="prof_label">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>20</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Sin usuario</string>
    </property>
   </widget>
   <widget class="QPushButton" name="label_title">
    <property name="geometry">
     <rect>
      <x>259</x>
      <y>-1</y>
      <width>261</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="text">
     <string>Diagrama</string>
    </property>
   </widget>
   <widget class="QLabel" name="num_of_act">
    <property name="geometry">
     <rect>
      <x>174</x>
      <y>5</y>
      <width>21</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#num_of_act
{
color: yellow;
border:none;
background-color: transparent;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="modulo_9">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>390</y>
      <width>181</width>
      <height>121</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">#modulo_9 { 
border: none; 
background-repeat: none;
background-position: center;
} </string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QWidget" name="widget_25" native="true">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>220</y>
      <width>10</width>
      <height>10</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="widget_26" native="true">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>180</y>
      <width>10</width>
      <height>10</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="widget_27" native="true">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>150</y>
      <width>10</width>
      <height>10</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="water_level">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>50</y>
      <width>81</width>
      <height>461</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#water_level{
background-image:url(:/images/images/watertile.png);
border:none;
background-position: bottom left;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <zorder>num_of_act</zorder>
   <zorder>top_bar</zorder>
   <zorder>bottom_bar</zorder>
   <zorder>pushButton</zorder>
   <zorder>pb_fisicos</zorder>
   <zorder>pb_quimicos</zorder>
   <zorder>pb_electricos</zorder>
   <zorder>active_param_label</zorder>
   <zorder>asa_logo</zorder>
   <zorder>label_hora</zorder>
   <zorder>label_dia</zorder>
   <zorder>prof_pic</zorder>
   <zorder>prof_label</zorder>
   <zorder>label_title</zorder>
   <zorder>modulo_9</zorder>
   <zorder>widget_27</zorder>
   <zorder>widget_26</zorder>
   <zorder>widget_25</zorder>
   <zorder>water_level</zorder>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
